fixes.inc

```diff
@@ -10184,23 +10259,23 @@ native BAD_SetPlayerPosFindZ(playerid, Float:x, Float:y, Float:z) = SetPlayerPos
 			switch (GetPlayerAnimationIndex(playerid))
 		#endif
 			{
-				case 958, 959, 960, 961, 962, 1134:
+			case 958, 959, 960, 961, 962, 1134:
```

Comment:
It's looks weird, isn't it? Why you has changed the indentation for all these lines?

Reply:
I didn't even notice that - let me look in to it.  Most of that (I think) was from the merge.  There should really only be a few minor changes in this pull request.